udata/core/metrics/commands.py

```diff
@@ -114,4 +115,18 @@ def update(site=False, organizations=False, users=False, datasets=False,
                 except Exception as e:
                     log.info(f'Error during update: {e}')
                     continue
+
+    if do_all or geozones:
```

Comment:
I think this is a punctual command to compute all explicitly. However, it isn't scheduled on contrary to `compute_site_metrics` task.

All `count_objects` methods here are instead triggered at runtime on object creation/update/deletion with signals.
Maybe we could apply the same logic as `update_datasets_metrics` for example?

Reply:
My point was that it won't get udpated at all if we deploy it as is (or am I missing something?). `update` command is not a scheduled job.
I agree about the questionable need about having real time metrics for this, it was just a suggestion to copy other metrics behavior for coherence.

Reply:
Yes I understood :-) I was wondering if I needed to implement the update of the metrics with the signals or with a job updating them every night :-)

Reply:
Done in [6556e07](https://github.com/opendatateam/udata/pull/3018/commits/6556e07bdaa2566b6e50ce316b46a2bae0e10f99)